charts/kuberpult/templates/cd-service.yaml

```diff
@@ -195,9 +231,33 @@ spec:
           value: "{{ .Values.datadogProfiling.enabled }}"
         - name: KUBERPULT_MAXIMUM_QUEUE_SIZE
           value: "{{ .Values.cd.backendConfig.queueSize }}"
+{{- if .Values.cd.db.enabled }}
+        - name: KUBERPULT_DB_ENABLED
+          value: "true"
+        - name: KUBERPULT_DB_OPTION
+          value: "cloudsql"
```

Comment:
there needs to be a way to set this.
We need 3 options: no db, cloudsql and postgres.